app/user/rest.py

```diff
@@ -133,6 +130,14 @@ def update_user_attribute(user_id):
     return jsonify(data=user_to_update.serialize()), 200
 
 
+def set_reply_to(recipient, template):
```

Comment:
This method name is not as precise as I think it should be. I think there needs to be something in the name to show that it is to do with setting the reply to for messages sent from the Notify service, rather than any generic service.

Also this method isn't `setting` anything, it's `getting` something.

I'd suggest `get_reply_to_for_notify_service`